thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/Utils.java

```diff
@@ -0,0 +1,42 @@
+package com.linkedin.thirdeye.anomalydetection;
```

Comment:
We already have a Utils class. You could either use that one itself, or name this to AnomalyUtils to be more specific?